build/common.sh

```diff
@@ -649,6 +649,50 @@ function kube::build::start_rsyncd_container() {
   kube::log::error "Could not connect to rsync container. See build/README.md for setting up remote Docker engine."
   return 1
 }
+# This will pack kube-system manifests files for distros without using salt
+# such as GCI and Ubuntu Trusty. We directly copy manifests from
+# cluster/addons and cluster/saltbase/salt. The script of cluster initialization
+# will remove the salt configuration and evaluate the variables in the manifests.
+function kube::release::package_kube_manifests_tarball() {
```

Comment:
The was a rebase error. After fixing it, it's a 3-line diff in the original function

I'll update the PR after verifying things still work..

Thanks for catching it!